### DISCUSSION TITLE
sparkline example with xychart?

### DISCUSSION THREAD

**Discussion by bonesoul** [airbnb/visx#1451] [Q&A]
as data-ui is now a frozen project, recommends using xychart component.

any examples for a sparkline implementation with it?

**Comment by williaster** [comment]
@bonesoul do you have a visual of what you are trying to achieve? a sparkline is usually a xychart with minimal axes/grid lines, but would like to check what you are going for before spending time on an example.

Converting this to a discussion since it's not a problem with the lib.

**Comment by bonesoul** [comment]
well actually trying to get a similar same look with your old data-ui, and with tooltips.

this one especially;

![image](https://user-images.githubusercontent.com/73215/157544901-56750bee-13fa-4017-a624-184aacedf56c.png)

**Reply by williaster** [reply]
hey @bonesoul sorry for the delay, crazy week. yes this should be straight forward! here's a [sandbox demo](https://codesandbox.io/s/sparkline-demo-ufd7nd?file=/Example.tsx:547-626) of using gradients with `@visx/xychart` + `@visx/gradient` (`@data-ui` re-exported the contents of that package)

![image](https://user-images.githubusercontent.com/4496521/157999757-5ca15bb0-0e2f-4ff5-bb34-09154784661c.png)

**Comment by bonesoul** [comment]
so is it possible to use LinearGradient with XYChart?
